lib/public/views/Runs/Overview/RunsOverviewModel.js

```diff
@@ -51,81 +52,59 @@ export class RunsOverviewModel extends Observable {
         this._eorReasonsFilterModel.observe(() => this._applyFilters());
         this._eorReasonsFilterModel.visualChange$.observe(() => this.notify());
 
-        this._pagination = new PaginationModel();
-        this._pagination.observe(() => this.fetchRuns());
-        this._pagination.itemsPerPageSelector$.observe(() => this.notify());
-
-        // Content
-        this._currentPageRuns = RemoteData.notAsked();
-        this._allRuns = RemoteData.notAsked();
+        // Exportable items
+        this._exportableDataSource = new PaginatedRemoteDataSource();
+        this._exportableItems = ObservableData.builder()
```

Comment:
For me it should simply be `exportItems`, because it's the items of the export.

Reply:
appied